src/kOS/Control/WheelSteeringManager.cs

```diff
@@ -0,0 +1,160 @@
+            controlPartId = 0;
+            Enabled = false;
+        }
+
+        void IFlightControlParameter.DisableControl(SharedObjects shared)
+        {
+            if (shared.KSPPart.flightID == controlPartId)
+                return;
+            ((IFlightControlParameter)this).DisableControl();
+        }
+
+        void IFlightControlParameter.EnableControl(SharedObjects shared)
+        {
+            controlPartId = shared.KSPPart.flightID;
+            controlShared = shared;
+            Enabled = true;
+        }
+
+        SharedObjects IFlightControlParameter.GetShared()
+        {
+            return controlShared;
+        }
+
+        object IFlightControlParameter.GetValue()
+        {
+            if (Enabled)
+            {
+                return Value;
+            }
+            return internalVessel.ctrlState.mainThrottle;
+        }
+
+        void IFlightControlParameter.UpdateAutopilot(FlightCtrlState c)
+        {
+            if (!Enabled) return;
+
+            if (!(controlShared.Vessel.horizontalSrfSpeed > 0.1f)) return;
+
+            if (Mathf.Abs(VesselUtils.AngleDelta(VesselUtils.GetHeading(controlShared.Vessel), VesselUtils.GetVelocityHeading(controlShared.Vessel))) <= 90)
+            {
+                c.wheelSteer = Mathf.Clamp(Value / -10, -1, 1);
+            }
+            else
+            {
+                c.wheelSteer = -Mathf.Clamp(Value / -10, -1, 1);
+            }
+        }
+
+        void IFlightControlParameter.UpdateState()
+        {
+            throw new NotImplementedException();
+        }
+
+        void IFlightControlParameter.UpdateValue(object value, SharedObjects shared)
+        {
+            if (!Enabled)
+                ((IFlightControlParameter)this).EnableControl(shared);
+
+            float bearing = 0;
+
+            if (value is VesselTarget)
+            {
+                bearing = VesselUtils.GetTargetBearing(controlShared.Vessel, ((VesselTarget)value).Vessel);
+            }
+            else if (value is GeoCoordinates)
+            {
+                bearing = ((GeoCoordinates)value).GetBearing();
+            }
+            else
+            {
+                try
+                {
+                    double doubleValue = Convert.ToDouble(value);
+                    if (Utils.IsValidNumber(doubleValue))
+                    {
+                        bearing = (float)(Math.Round(doubleValue) - Mathf.Round(FlightGlobals.ship_heading));
+                        if (bearing < -180)
+                            bearing += 360; // i.e. 359 degrees to the left is really 1 degree to the right.
+                        else if (bearing > 180)
+                            bearing -= 360; // i.e. 359 degrees to the right is really 1 degree to the left
+                    }
+                }
+                catch
+                {
+                    throw new KOSWrongControlValueTypeException(
+                        "WHEELSTEER",
```

Comment:
The name used in kerboscript is wheelsteerING - this message may confuse users if it refers to it as just WHEELSTEER.